### ISSUE TITLE
In TinyMCE edit a link target blank and No Follow switch back to Alternate

### ISSUE THREAD

**Issue by AlexRed** [tinymce/tinymce#3659]
**Do you want to request a *feature* or report a *bug*?**
I want to report a bug.

**What is the current behavior?**
In a new article insert a text and with TinyMCE insert a link target blank and No Follow, try to edit the link and it appears in the TinyMCE window like Rel Alternate

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.

![link-editor3](https://cloud.githubusercontent.com/assets/1362388/26147035/274f897e-3aea-11e7-8312-974f3ca9bb8a.gif)

**What is the expected behavior?**
edit a link target blank and No Follow no switch back to Alternate

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
TinyMCE 4.5.6
It is work in previous versions.

**Comment by valse** [comment]
Hi,
   this is the same my issue https://github.com/tinymce/tinymce/issues/3650.

I think that it's only an edit wrong behavior because, if you check the html source, you should see "nofollow" rel attribute again.

**Comment by AlexRed** [comment]
yes, like I write: "it appears in the TinyMCE window like Rel Alternate"
but in the code it is rel="nofollow noopener noreferrer" and you can't set it "Alternate"

**Comment by jayarjo** [comment]
@AlexRed do you also have custom `rel_list` in your TinyMCE config?

**Comment by AlexRed** [comment]
Hello @jayarjo, thanks for your reply. How can I test if I have custom rel_list in Joomla 3.7.1 TinyMCE config ?  Where can I find it ?
If you want test I setup a test site:
http://tinymce.freesite.host
user: tinymce
password: tinymce

**Comment by jayarjo** [comment]
It seems, like Joomla has it defined internally. I've scheduled this for a fix. Thanks for reporting @AlexRed, @valse.

**Comment by AlexRed** [comment]
in  Joomla_3.7.1/plugins/editors/tinymce/tinymce.php  
I found:
![tinymce php](https://cloud.githubusercontent.com/assets/1362388/26281938/b0570ee8-3dfd-11e7-9224-f66794cbc9f6.png)

**Comment by Maniraj-Vanna** [comment]
Is the issue fixed? @AlexRed .. I am getting the same issue here https://repl.it/@ManirajVanna/tinymcevanna#index.html where i am using tinymce version 5..

**Comment by sagun786** [comment]
Has this been fixed ?

**Comment by EzVortex** [comment]
Well guys, just worked with some legacy project and spend a couple hours on this

Check **valid_elements** option in your init config, if it contains something like **a[href|class|target]** which validates all links and attributes, then just add **rel** in the end as additional attribute.

![изображение](https://github.com/tinymce/tinymce/assets/50105426/d7735fa9-31ae-4241-a99b-7b4ea3a6f535)

**Comment by kk-11** [comment]
This behaviour appears to be intentional and for security.

You can use `allow_unsafe_link_target: true`

https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#allow_unsafe_link_target
